### DISCUSSION TITLE
Microsoft Fluent Emoji as default (it is open source)

### DISCUSSION THREAD

**Discussion by StigtriX** [florisboard/florisboard#2170] [Feedback]
Microsoft's Fluent Emoji are open source and look so much better than everything else.

Can this please become the standard for FlorisBoard, or at least an option?

**Link to GitHub:**
https://github.com/microsoft/fluentui-emoji

![image](https://user-images.githubusercontent.com/58557127/224294064-4667911b-353b-4084-a99f-0c9399aec6de.png)

**Comment by patrickgold** [comment]
Thanks for your feature request!

The problem with emoji fonts is that it is not really the power of FlorisBoard to control the emoji font but the systems (or nowadays increasingly per-app). Surely I could use Microsoft's Fluent Emoji font to render the emojis within the emoji picker, but in the app you are using they are still in the emoji font what the system or app desires.

As such FlorisBoard tries to communicate with the app to see if it uses a specific EmojiCompat font, if so FlorisBoard uses that. Else it just uses the system emoji design. Hope I could explain this without going too much into technical details behind this.

~Closing this issue as wont-fix.~ edit: this isn't an issue so can't close it but I still properly labeled it.
